README.md

```diff
@@ -34,3 +34,5 @@ is loaded.
 - **ipam\_get\_vlans**: Get VLAN(s) via optional parameters
 - **ipam\_get\_vrfs**: Get VRF(s) via optional parameters
 - **ipam\_get\_prefixes**: Get Prefix(es) via optional parameters
+- **ipam\_get\_available_ips**: Get available IP Address(es) within a prefix
+- **ipam\_post\_available_ips**: Get available IP Address(es) within a prefix via optional parameters
```

Comment:
Please change this description to reflect this being a POST to assign the first available IP address in the given prefix.